homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import {inject, observer} from 'mobx-react';
+import Todo from "./component/todo.js";
+import Add from "./component/add";
+import Update from "./component/update";
+
+@inject('myTodoApp')
+@observer
+
+class App extends Component {
+
+  render() {
+    const {myTodoApp} = this.props;
+
+    const isEmpty = myTodoApp.data.todos.length < 1 || myTodoApp.data.todos === undefined;
+
+    const todoList = myTodoApp.data.todos.map((item, key)=> 
```

Comment:
What is the value of `key` here?
`id` is for identifying, while index for what it means, don't we want to identify ? if yes 😃 then we may use `item.id` instead of `key` here?

Reply:
Actually I added `key` because of I got an error. But now I removed everything I did not used. It looks prettier. I am using just `id={item._id}` for iterator and identification